tee-worker/litentry/core/vc-task/receiver/src/test.rs

```diff
@@ -0,0 +1,105 @@
+use core::time::Duration;
+
+use lc_vc_task_sender::{VCRequest, VcRequestSender};
+use litentry_primitives::AesRequest;
+
+use super::*;
+use crate::mock::*;
+
+#[test]
+#[ignore]
```

Comment:
Why is this test case ignored ?

Reply:
Same reason as above

Reply:
Don't know why this duplicated comment was added :)